packages/terra-consumer-icon/scripts/src/generate-example/parseCsv.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
also all the same right?
https://github.com/cerner/terra-core/blob/master/packages/terra-icon/scripts/src/generate-example/parseCsv.js